app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
@tenderlove We have similar current controller/model setup in other parts of the app with controller driving record deletion.